cmd/cluster/context.go

```diff
@@ -393,6 +400,25 @@ func (o *contextOptions) generateContextData() (*contextData, []error) {
 		}
 	}
 
+	GetDynatraceURL := func() {
+		defer wg.Done()
+		if o.verbose {
+			fmt.Fprintln(os.Stderr, "Getting Dynatrace URL...")
+		}
+		if !strings.HasPrefix(cluster.Name(), "hs-mc-") {
+			data.DyntraceEnvURL = "cluster is Not a Management Cluster"
```

Comment:
If it's a HCP cluster, can we also return an dynatrace URL on the correct tenant with  some filter preset so it can save our DDU when doing the query? like we can append the filter log type, management cluster name, namespace name and status error as a default.

Reply:
For HCP, what we can do can be:
1. Check it's HCP
2. Find it's management cluster
3. Build k8s client to the mangement cluster and query the dynatrace CRD

Reply:
@wanghaoran1988 This will be taken up in https://issues.redhat.com/browse/OSD-19655.